### ISSUE TITLE
PyCon 2015 Videos

### ISSUE THREAD

**Issue by SylvainDe** [s16h/py-must-watch#46]
I'm only guessing but there should be a bunch of new interesting videos coming with PyCon 2015.

I'm only assuming that this https://www.youtube.com/watch?v=wf-BqAjZb8M is going to be as awesome as the other presentations from Raymond Hettinger .

**Comment by s16h** [comment]
Yes yes. Planning to add the PyCon 2015 must-watches soonsest. :)

**Comment by s16h** [comment]
Just watched it; it's awesome. Adding it now.

**Comment by SylvainDe** [comment]
Well done!
Btw: here's the presentation https://us.pycon.org/2015/schedule/presentation/416/ . I couldn't find the slides yet.

**Comment by SylvainDe** [comment]
Also (I'll handle this as soon as I have some more time) : 

http://nedbatchelder.com/text/names.html
http://nedbatchelder.com/text/names1.html
http://nedbatchelder.com/text/names1/names1.html
https://www.youtube.com/watch?v=_AEJHKGk9ns
http://pyvideo.org/video/3466/facts-and-myths-about-python-names-and-values
https://us.pycon.org/2015/schedule/presentation/362/
